src/core/public/chrome/chrome_service.tsx

```diff
@@ -465,6 +537,16 @@ export interface ChromeStart {
    */
   setIsVisible(isVisible: boolean): void;
 
+  /**
+   * Get an observable of the current header variant.
+   */
+  getHeaderVariant$(): Observable<HeaderVariant | undefined>;
+
+  /**
+   * Set or unset the temporary variant for the header.
+   */
+  setHeaderVariant(variant?: HeaderVariant): void;
```

Comment:
Is there value to calling this without an argument? Or does that happen through some code path that requires this?

Reply:
`undefined` is to unset an override.